content/en/docs/File Format/implementationstatus.md

```diff
@@ -22,91 +22,91 @@ Implementations:
 * `Go`: [parquet-go](https://github.com/apache/arrow-go/tree/main/parquet)
 * `Rust`: [parquet-rs](https://github.com/apache/arrow-rs/blob/main/parquet/README.md)
 * `cuDF`: [cudf](https://github.com/rapidsai/cudf)
-
+* `JavaScript`: [hyparquet](https://github.com/hyparam/hyparquet)
 
 
 ### Physical types
 
-| Data type                                 | C++   | Java  | Go    | Rust  | cuDF  |
-| ----------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| BOOLEAN                                   |  ✅   |  ✅   |       |  ✅   |  ✅   |
-| INT32                                     |  ✅   |  ✅   |       |  ✅   |  ✅   |
-| INT64                                     |  ✅   |  ✅   |       |  ✅   |  ✅   |
-| INT96 (1)                                 |  ✅   |  ✅   |       |  ✅   |  ✅   |
-| FLOAT                                     |  ✅   |  ✅   |       |  ✅   |  ✅   |
-| DOUBLE                                    |  ✅   |  ✅   |       |  ✅   |  ✅   |
-| BYTE_ARRAY                                |  ✅   |  ✅   |       |  ✅   |  ✅   |
-| FIXED_LEN_BYTE_ARRAY                      |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| Data type                                 | C++   | Java  | Go    | Rust  | cuDF  | JS  |
```

Comment:
```suggestion
| Data type                                 | C++   | Java  | Go    | Rust  | cuDF  | hyparquet |
```

Same for below.

Reply:
updated tables with library name "hyparquet"